root/Makefile

```
@@ -35,14 +35,30 @@ BASE_TEST_ENV       ?= GOPROXY=$(GOPROXY) GOPRIVATE=$(GOPRIVATE) OUTREACH_ACCOUN
 .PHONY: default
 default: build
 
+# All the pre-action steps are bunched together here.
+.PHONY: pre-release pre-build pre-test pre-coverage pre-integration pre-e2e pre-benchmark pre-gogenerate pre-devserver pre-debug pre-docker-build pre-fmt
+pre-release::
+pre-build::
+pre-test::
+pre-coverage::
+pre-integration::
+pre-e2e::
+pre-benchmark::
+pre-gogenerate::
+pre-devserver::
+pre-debug::
+pre-docker-build::
+pre-fmt::
+
+
 ## release:         tag a new release with goreleaser
 .PHONY: release
-release:
+release:: pre-release
 	@# Create a tag for our version
 	@git tag -d "$(APP_VERSION)" >&2 || true
 	@git tag "$(APP_VERSION)" >&2
 	@./.bootstrap/shell/gobin.sh github.com/goreleaser/goreleaser@v0.157.0 release --skip-publish --rm-dist
-	@# Delete the tag once we're done.
+	@# Delete the tag once we\'re done.
```

That really needs to be escaped? If so I'd rather just avoid it entirely.
```suggestion
	@# Delete the tag once we are done.
```